apps.toml

```diff
@@ -142,6 +142,7 @@ url = "https://github.com/YunoHost-Apps/anarchism_ynh"
 
 [archivebox]
 added_date = 1674232499 # 2023/01/20
+antifeatures = [ "package-not-maintained" ]
```

Comment:
I'm a bit confused, both the upstream and the package seem somewhat active / maintained ?

Reply:
If you disagree, I can accept that entirely, but allow me to build my case:
Upstream is absolutely active and maintained, but I believe that the yunohost package is undermaintained.
The yunohost package has not received attention for 3 months, and I believe that the last time it was updated to match version 0.7.2, it was done improperly. [Please see my pull request](https://github.com/YunoHost-Apps/archivebox_ynh/pull/37) [and my issue](https://github.com/YunoHost-Apps/archivebox_ynh/issues/36).
I also believe that it otherwise needs attention to polish missing or nonfunctioning pull methods.

a version of archivebox installed on my ubuntu (non-yunohost) machine using the bash setup script (without docker)
![Screenshot from 2024-05-29 21-22-30](https://github.com/YunoHost/apps/assets/54383546/c569fa3a-3ce2-4445-8893-cbb4647d230d)
Please note that the internet archive rarely works (works 10% of the time)

A version of archivebox using this yunohost installation
![Screenshot from 2024-05-29 21-26-13](https://github.com/YunoHost/apps/assets/54383546/994fb38f-957e-4ece-a074-5e064a8b07e5)
Please note the 0.6.2 watermark, which is a version that is 4 years old. Singlefile is nonfunctional.